src/ServiceBuilder.php

```diff
@@ -0,0 +1,30 @@
+<?php
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud;
+
+use Google\Cloud\Core\ServiceBuilder as CoreServiceBuilder;
+
+/**
+ * This class is here to maintain backwards compatibility with previous
+ * versions.
+ *
+ * @deprecated use Google\Cloud\Core\ServiceBuilder instead
+ */
+class ServiceBuilder extends CoreServiceBuilder
+{
```

Comment:
To maintain backwards compatibility we may need to keep the version constant in this class as well.